include/spirv/unified1/spirv.core.grammar.json

```
@@ -16688,6 +16737,12 @@
           "extensions" : [ "SPV_QCOM_tile_shading" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "CooperativeMatrixConversionQCOM",
+          "value" : 4496,
+          "extensions" : [ "SPV_QCOM_cooperative_matrix_conversion" ],
```

misses the implicit declaration

```suggestion
          "capabilities" : [ "CooperativeMatrixKHR" ],
          "extensions" : [ "SPV_QCOM_cooperative_matrix_conversion" ],
```

---

Ah, Thank you for spotting it. Fixed.